frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -50,25 +47,24 @@ const getPluginTabPages = (tabs: DashboardsTab[], cards: DashboardsCard[]): Page
 const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight, k8sModels }) => {
   const { t } = useTranslation();
   const title = t('dashboard~Overview');
+
+  /* To add a new tab next to cluster overview */
+  /*
   const tabExtensions = useExtensions<DashboardsTab>(isDashboardsTab);
   const cardExtensions = useExtensions<DashboardsCard>(isDashboardsCard);
-
   const pluginPages = React.useMemo(() => getPluginTabPages(tabExtensions, cardExtensions), [
     tabExtensions,
     cardExtensions,
   ]);
+  */
 
-  const allPages = React.useMemo(
-    () => [
-      {
-        href: '',
-        name: t('dashboard~Cluster'),
-        component: ClusterDashboard,
-      },
-      ...pluginPages,
-    ],
-    [pluginPages, t],
-  );
+  const allPages = [
+    {
+      href: '',
+      name: t('dashboard~Cluster'),
+      component: ClusterDashboard,
+    },
+  ];
```

Comment:
Please revert this change as per https://github.com/openshift/console/pull/8496/files#r609376762

Reply:
got it... will revert the changes...